coc/war_clans.py

```diff
@@ -147,6 +147,15 @@ def defenses(self) -> typing.List["WarAttack"]:
         """
         other = self._war and self._war.clan if self.is_opponent else self._war.opponent
         return other.attacks if other else []
+    
+    @property
+    def average_attack_duration(self) -> int:
+        """:class:`int`: Returns the average duration of all clan member's attacks this war."""
+        total_duration = 0
+        count = len(self._cs_attacks)
+        for attack in self._cs_attacks:
+            total_duration += attack.duration
+        return round(total_duration/count,0) if count > 0 else 0
```

Comment:
Since the docstring/signature of this function says it returns an int, it may be better to use `int(total_duration / count)` instead of round, since round will return a float even if you specify 0 decimal places - 

![image](https://user-images.githubusercontent.com/38833292/129820811-99b775e6-1f63-4b57-b6f5-e3f8880d2c54.png)